async/compiler/src/openxla/compiler/Dialect/Async/Transforms/test/async_to_runtime.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: iree-opt --iree-plugin=openxla-async --split-input-file \
+// RUN:   --async-to-async-runtime | \
+// RUN: FileCheck %s
+
+func.func @await_token(%arg0: !async.token){
+  // CHECK: call @async.value.await.token
+  async.await %arg0 : !async.token
+  return
+}
+
+// CHECK:  func.func @await_token(%[[ARG:.*]]: !async.value) {
```

Comment:
nit: indentation is slightly off, just one space between CHECK: and the first line

Reply:
Fixed